tests/cases/file_test.py

```diff
@@ -781,6 +784,18 @@ def testFilesystemAssetstore(self):
         copyTestFile = self._testUploadFile('helloWorld1.txt')
         self._testCopyFile(copyTestFile)
 
+        # Test unicode filenames for content disposition.  The test name has
+        # quotes, a Linear-B codepoint, Cyrllic, Arabic, Chinese, and an emoji.
+        filename = u'Unicode "sample" \U00010088 ' + \
```

Comment:
I think it'd be a good idea to put this test string somewhere general, and start using it as test data for any place that should be expected to handle general strings (which includes Unicode, quotes, etc.) gracefully.

There is some risk of "overfitting" to this one test string, but if we think of new corner cases that challenge string handling, we can just add them to the master string.

Reply:
We probably should have a set of test strings and one master complex string like this one.  I could see a pure right-to-left string giving different results than a mixed direction string, for instance.